crates/sui-graphql-rpc/src/types/move_object.rs

```diff
@@ -34,9 +37,17 @@ impl MoveObject {
         Some(MoveValue::new(type_, self.native.contents().into()))
     }
 
-    /// Determines whether a tx can transfer this object
-    async fn has_public_transfer(&self) -> Option<bool> {
-        Some(self.native.has_public_transfer())
```

Comment:
i see, looks like this field has been deprecated since about 2 months ago with https://github.com/MystenLabs/sui/pull/14445